### ISSUE TITLE
Summary tab has inconsistent scrolling

### ISSUE THREAD

**Issue by dbeyer** [sosy-lab/benchexec#491]
The Summary tab consists of two parts:
- the details of the benchmark executions (top) and
- statistics for for the run sets (bottom).

The top table is taking the width that it needs, while the bottom table
takes the width of my display port and adds a local scroll bar.

For horizontal scrolling of the top table, I have to use the scroll bar for the whole window
to scroll to the right, while for the bottom table, I have to use the local scroll bar.
In other words, to scroll the top table, I need to use the scroll bar at the very bottom.

I am not sure what is the rational behind this design.
I guess I would prefer to either have the bottom table also take the width it needs
and I scroll only with the scroll bar of the window,
or put both together in a table with a local scroll bar.

A side topic:
Is `Environment` the best name? Perhaps `Benchmark Setup` or `Execution Details` is better?

**Comment by PhilippWendler** [comment]
The reason that the bottom table has its own scroll bar is the "Fixed row title" feature. This is quite useful, in the old table one would always loose orientation in the statistics part about which line means what. So I think this feature should stay.

That leaves two options:
- keep as is
- merge both tables and let it have a local scroll bar

**Comment by PhilippWendler** [comment]
It seems that the library that we use for the statistics table (which is the same as for the main table) does not support cells that arbitrarily span several columns. This is needed for the "Environment" table (when comparing several run sets, it highlights the fact whether they were executed in the same environment or not).

So summary of currently known options:
- keep as is
- merge tables and loose column spanning of environment (drawback explained in this comment)
- merge tables and loose "Fixed row title" (drawback explained in https://github.com/sosy-lab/benchexec/issues/491#issuecomment-549327156)
- find a new table library (at least for the summary tab) that supports both

**Comment by PhilippWendler** [comment]
I added a local scroll bar to the "Environment" table. This fixes the most important part of this issue (the inconsistent scrolling mentioned in the issue title). Whether and how to merge the two tables on the summary tab is a much more complex topic that would better be discussed in a separate issue if still desired.

And btw., I renamed both tables to "Benchmark Setup" and "Statistics", respectively.
